wom/src/main/scala/wom/graph/InstantiatedExpression.scala

```diff
@@ -19,26 +15,10 @@ object InstantiatedExpression {
     val graphNodeSetter = new GraphNode.GraphNodeSetter()
 
     for {
-      linkedInputs <- InstantiatedExpression.linkWithInputs(graphNodeSetter, expression, inputMapping)
+      linkedInputs <- expression.linkWithInputs(graphNodeSetter, inputMapping)
       expressionNode = nodeConstructor(nodeIdentifier, linkedInputs)
       _ = graphNodeSetter._graphNode = expressionNode
     } yield expressionNode
   }
 
-  def linkWithInputs(graphNodeSetter: GraphNode.GraphNodeSetter, expression: WomExpression, inputMapping: Map[String, OutputPort]): ErrorOr[InstantiatedExpression] = {
-    def linkInput(input: String): ErrorOr[(String, InputPort)] = if (inputMapping.contains(input)) {
-      val upstreamPort = inputMapping(input)
-      Valid((input, ConnectedInputPort(input, upstreamPort.womType, upstreamPort, graphNodeSetter.get)))
-    } else {
-      s"Expression cannot be connected without the input $input (provided: ${inputMapping.toString})".invalidNel
-    }
-
-    import lenthall.validation.ErrorOr.ShortCircuitingFlatMap
-    for {
-      linkedInputList <- expression.inputs.toList traverse linkInput
-      linkedInputs = linkedInputList.toMap
-      inputTypes = linkedInputs map { case (k, v) => k -> v.womType }
-      evaluatedType <- expression.evaluateType(inputTypes)
-    } yield new InstantiatedExpression(expression, evaluatedType, linkedInputs)
-  }
 }
```

Comment:
While you're here, might as well add a newline

Reply:
I'm not trying to start a bike shed discussion, but why does it matter?

Reply:
It's a @kshakir hobby horse because if you cat a file it doesn't put the prompt on its own line.

something something autoformatting something something